test/main/accounts/index.test.js

```diff
@@ -780,3 +791,54 @@ describe('#resolveRequest', () => {
     expect(Object.keys(Accounts.current().requests)).toHaveLength(0)
   })
 })
+
+describe('#removeRequest', () => {
+  beforeEach(() => {
+    store.navClearReq = jest.fn()
+    account.update = jest.fn()
+  })
+
+  it('should remove a request for the provided handlerId from the account object', () => {
+    Accounts.addRequest(request)
```

Comment:
this can be moved into a `beforeEach` hook since it really defines the pre-test state and isn't a part of the test itself